activities.json

```diff
@@ -263,6 +263,18 @@
     "title": "Credential Management Level 1",
     "url": "https://w3c.github.io/webappsec-credential-management/"
   },
+  {
+    "ciuName": null,
+    "description": "Blocklist certain opaque responses based on MIME type and return an 'emptied' response instead.",
+    "id": "corb",
+    "mozBugUrl": "https://bugzilla.mozilla.org/show_bug.cgi?id=1459357",
+    "mozPosition": "non-harmful",
+    "mozPositionDetail": "While this is an important aspect of a robust Spectre-defense, we would like to see a safelist-based approach pursued.",
```

Comment:
I think it's probably worth explicitly pointing to (or at least naming) the alternative proposal that you're working on.  Since if we're saying we prefer an alternative, we should do in a way that it's clear to readers that we're talking about an alternative that's actually being developed (rather than, say, a proposed change to this spec).

Reply:
Would that require some changes to make mozPositionDetail markdown or some such?

Reply:
Currently `mozPositionDetail` accepts arbitrary HTML, so no changes needed!

Reply:
Turns out that we should look out for markup more carefully. 😊